runconfig/parse.go

```diff
@@ -65,6 +65,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flCpuShares       = cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
 		flCpusetCpus      = cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems      = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flBlkioWeight     = cmd.Int64([]string{"-blkio-weight"}, 0, "Block IO relative weight, range from 10 to 1000")
```

Comment:
Heh, that's a way of looking at it, yes. No problem in this case, but it will make it impossible to distinguish between "not specified" and "value is set" if `0` is a valid value.

Anyway, now we're bike-shedding, you're right that the same approach is taken in other places, so for consistency, it's probably best this way (but personally....)